.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'Type: Bug'
+assignees: ''
+
+---
+
+**Your environment**
```

Comment:
I think we can be more specific here and include the following:

- OS
- Commit hash that you are currently running on
- Version of Ruby
- Version of Rails